api/.project

```diff
@@ -5,19 +5,13 @@
 	<projects>
```

Comment:
remove this file entirely

Reply:
Sorry that I forgot, I'll remove it next commit (Waiting to see if you have other comments on file to remove)